app.yaml

```diff
@@ -10,3 +10,8 @@ handlers:
   static_files: build/index.html
   upload: build/index.html
   secure: always
+  http_headers:
+    X-Frame-Options: "DENY"
+    Strict-Transport-Security: "max-age=2592000; includeSubdomains"
+    X-Content-Type-Options: "nosniff"
+    X-XSS-Protection: "1; mode=block"
```

Comment:
Anything else we should have in [`Content-Security-Policy`](https://infosec.mozilla.org/guidelines/web_security#content-security-policy)?

Nit: Also, end with a newline, plz.

Reply:
👍